### PR TITLE
Prepare basic project structure Create CircelCi Configuration  Issue#2

### LINKED ISSUE #2: Prepare basic project structure Create CircelCi Configuration
- Prepare circle CI configuration
- Prepare AWS ECS setup
- Clone and install VueJS template and prepare it for work
- Prepare the login/logout mechanism

### PR DESCRIPTION
**Pull request by devahmedshendy**
implement of issue
Fixes #2

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+      - run:
+          name: Build The Project - Production Version
+          command: npm run build
+          working_directory: ~/beestock-admin-dashboard/vuejs-app
+
+      - add_ssh_keys:
+          fingerprints:
+            - "20:7f:8b:cd:93:f6:07:61:f2:b2:4e:46:f8:b3:a1:23"
+
```
Comment (almasry) [inline .circleci/config.yml]: add comment documentation for each block of these instructions

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.9.1
+    braches:
+      - only: master
```
Comment (almasry) [inline .circleci/config.yml]: ???

Comment (almasry) [reply .circleci/config.yml]: JUST CHECK that the build is on each push, not every commit - if we have 10 commits locally and pushed them to the origin branch, it should build only once.

File: vuejs-app/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+COPY . /app
+
+EXPOSE 8080
+
+RUN npm install http-server -g
+
+RUN npm install && npm run build
+
+CMD http-server ./dist
```
Comment (almasry) [inline vuejs-app/Dockerfile]: ADD TODO : //  Test this file and try to use it in development mode!

and make changes to move it under   ./infrastructure/development

File: vuejs-app/LICENSE [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
```
Comment (almasry) [inline vuejs-app/LICENSE]: remove this bullshit.

File: vuejs-app/Dockerfile (hunk 3 above)
Comment (devahmedshendy) [reply vuejs-app/Dockerfile]: Ad TODO card is created for it [card-link](https://github.com/esna-technlogies/beestock-admin-dashboard/projects/1#card-6772958)

**Pull request merged**